io/plugins/eu.esdihumboldt.hale.io.xls.test/src/eu/esdihumboldt/hale/io/xls/test/reader/XLSReaderTest.java

```diff
@@ -314,6 +338,11 @@ private Schema readXLSSchema(String sourceLocation, int sheetIndex, String typeN
 		return schemaReader.getSchema();
 	}
 
+	private InstanceCollection readXLSInstances(String sourceLocation, int sheetIndex,
+			String typeName, boolean skipN, Schema sourceSchema) throws Exception {
+		return readXLSInstances(sourceLocation, sheetIndex, typeName, skipN ? 1 : 0, sourceSchema);
```

Comment:
This does not really check the backwards compatibility as parameter is passed to the reader as integer.

Reply:
This is tricky for me. I spent a lot of time being able to pass through those tests and I finally came up with this solution. In this way, I can pass to the method readXLSInstances (see lines 140 and 142) a boolean value for skipN that then is converted here into an integer. Does it not check backward compatibility in this way? Why not? 

Can you maybe suggest to me a possible solution (pseudo code)?

Reply:
I couldn't let your suggestion to use `Value skipN `instead of `boolean` in the definition of the method work fine.